dart/lib/model/binary_content_type.dart

```diff
@@ -0,0 +1,54 @@
+part of keyclic_sdk_api.api;
+
+class BinaryContentType {
```

Comment:
useless class without any properties

Reply:
This is an issue.
contentType of Binary must be a string.